core/src/test/java/com/google/zxing/oned/Code128WriterTestCase.java

```diff
@@ -151,4 +155,86 @@ private void testEncode(String toEncode, String expected) throws Exception {
     String actualRoundtripResultText = rtResult.getText();
     assertEquals(toEncode, actualRoundtripResultText);
   }
+
+  @Test
```

Comment:
For these, you can write `@Test(expected = IllegalArgumentException.class)` I'm not worried about the message

Reply:
That's neat!